numpy/add_newdocs.py

```diff
@@ -547,10 +547,26 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
-add_newdoc('numpy.core', 'broadcast', ('nd',
+add_newdoc('numpy.core', 'broadcast', ('ndim',
     """
     Number of dimensions of broadcasted result.
 
+    .. versionadded:: 1.12.0
+
```

Comment:
Projects like scipy support ancient versions of numpy, so it will be many, many years before ndim can replace nd, if ever. For that reason it is best to make clear the relationship between the two.